### PR TITLE
Create Tests for Pagination

### PR DESCRIPTION
**Pull request by leonli33**
Created unit tests that tests the pagination functionality of the participant manager grid.

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -141,6 +141,9 @@ export type ParticipantAccountSummary = {
   externalIds: StringDictionary<string>
   studyExternalId?: string
   status: 'unverified' | 'pending' | 'verified'
+  externalId?: string
```
Comment (conundrumgirl) [inline src/types/types.ts:144]: I don't think we need this here. This should map to what we get from the API

File: src/components/studies/participants/ParticipantSearch.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ const useStyles = makeStyles(theme => ({
 }))
```
Comment (conundrumgirl) [inline src/components/studies/participants/ParticipantSearch.tsx:72]: Did you make any changes in this file or is it just formatting?

**Review by conundrumgirl — CHANGES_REQUESTED** [review]
Nice work on the tests!  One question and one possible correction.

File: src/components/studies/participants/ParticipantSearch.tsx (hunk 2 above)
Comment (leonli33) [reply src/components/studies/participants/ParticipantSearch.tsx:72]: Just formatting

File: src/types/types.ts (hunk 1 above)
Comment (leonli33) [reply src/types/types.ts:144]: Gotcha. It appears that some results from the API do have an externalID field while others do not:
Screen Shot 2021-03-04 at 9 24 18 AM
v.s.
Screen Shot 2021-03-04 at 9 24 40 AM

I will remove it.

**Review by conundrumgirl — APPROVED** [review]
look good

**Pull request merged**